src/cli/api_client.rs

```diff
@@ -229,6 +229,7 @@ impl ApiClient {
         batch_group_name: String,
         on_chain_address: String,
         satoshis: u64,
+        metadata: serde_json::Value,
```

Comment:
shouldn't this be `Optional?

Reply:
Fixed https://github.com/GaloyMoney/bria/pull/121/commits/9cf6859da3a09a7cd8672f58de8754f4b7cd8546